rest-service/manager_rest/test/infrastructure/test_configure_manager.py

```diff
@@ -329,11 +364,37 @@ def test_update_manager_cert_differs(self):
         db.session.add(mgr)
 
         with self.assertRaisesRegex(RuntimeError, 'ca_cert.*differ'):
-            configure({
+            configure(user_config({
                 'manager': {
                     'hostname': 'mgr1',
                     'private_ip': 'example2.com',
                     'ca_cert': 'cert-content-2',
                 },
-            })
+            }))
         assert mgr.private_ip == 'example2.com'
+
+    def test_create_config_defaults(self):
+        configure(user_config({
+            'manager': {
+                'public_ip': 'example.org',
+            },
+            'mgmtworker': {
+                'max_workers': 9,
+            },
+            'agent': {
+                'heartbeat': 99
+            }
+        }))
+        created_config = db.session.scalars(select(models.Config)).all()
+        assert len(created_config) > 0
+
+        for scope, name, value in [
+            ('rest', 'public_ip', 'example.org'),
+            ('rest', 'file_server_url', 'https://localhost:53333/resources'),
+            ('mgmtworker', 'max_workers', 9),
+            ('agent', 'heartbeat', 99),
+        ]:
+            inst = db.session.scalars(
+                select(models.Config).filter_by(scope=scope, name=name)
+            ).first()
```

Comment:
```suggestion
            ).one()
```
if there's no entry, that should be already an error